awx/main/models/ha.py

```diff
@@ -289,6 +294,8 @@ def on_instance_group_saved(sender, instance, created=False, raw=False, **kwargs
     if created or instance.has_policy_changes():
         if not instance.is_containerized:
             schedule_policy_task()
+    elif created or instance.is_containerized:
```

Comment:
Yea, the seems pretty reasonable.

It might be nice to add a test to this file https://github.com/ansible/awx/blob/devel/awx/main/tests/functional/test_instances.py that:

1.  Creates an instance group and sets meaningful values to these attributes: https://github.com/ansible/awx/pull/5233/files#diff-4465e892d9e12c8fc652dc6f3e0de9cbR274-R276

2.  Assigns a k8s credential to the instance group (so that `is_containerized` becomes true).

3.  Verifies that after the `instance_group.save()` call, the values are unset to the defaults above.

Reply:
done, but in a different file, it was easier to have it there for writing tests, but I can move it if you think it would be better in the file you mentioned.